docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -38,32 +40,34 @@ used directly using native Kubernetes resource specifications.
 
 
 {% accordion setup %}
-  {% collapsible (Optional) Extract Kubernetes Specifications %}
-If hosting a private `helm` repository or using native Kubernetes specifications,
-extract the Kubernetes specifications required to deploy Alluxio from the Docker image.
```

Comment:
This section title was vague, and also not-optional if using `kubectl` directly. I've separated the use-cases of hosting a private Helm chart and fetching/generating templated YAMLs.